scipy/stats/tests/test_qmc.py

```diff
@@ -445,63 +457,50 @@ class TestHalton(QMCEngineTests):
 class TestLHS(QMCEngineTests):
     qmce = qmc.LatinHypercube
     can_scramble = False
-    unscramble_nd = np.array([[0.73412877, 0.50416027],
```

Comment:
Really not sure about this one... It was deterministic before with fixing the seed. If this is due to changing to using `uniform` then I would actually prefer to have the previous version.

Reply:
The reference value is implementation-dependent. Both 
```python
np.random.shuffle(np.arange(1, n+1, 1)) - np.random.uniform(n) / 2
```
and 
```python
np.random.permutation(n) + np.random.uniform(n) / 2
```
give 1-d LHS's but their values are different. It is not sensible IMO to re-write the test code every time there is a modification in the implementation. So I believe it is better to test without pre-computed samples.

Reply:
I am still confused that it worked with `rng.random`. But ok for me, the other tests are far more important in the end.